Cargo.toml

```diff
@@ -1,6 +1,5 @@
 [workspace]
 members = ["rust/crd", "rust/operator-binary"]
 
-# [patch."https://github.com/stackabletech/operator-rs.git"]
-# stackable-operator = { path = "/home/sbernauer/stackabletech/operator-rs" }
-# stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feat/affinities" }
+[patch."https://github.com/stackabletech/operator-rs.git"]
+stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feature/resource-quotas" }
```

Comment:
This can't be merged (maybe you already planned to fix this? I guess https://github.com/stackabletech/operator-rs/pull/598 needs to be merged first?)

Reply:
Patch removed and dependency updated in 23a0e62